lib/wab/impl/handler.rb

```diff
@@ -52,7 +52,7 @@ def log_request(caller, path, query)
       end
 
       def log_request_with_body(caller, path, query, body)
-        @shell.logger.info("#{caller}(#{path.join('/')}#{query}, #{body.json})")
+        @shell.logger.info("#{caller} #{path.join('/')}#{query}\n#{body.json(@shell.indent)}")
```

Comment:
with this we will have 3 lines that involve `.json(@shell.indent)`
would it be better if we extracted that to a private helper?
```ruby
prvate
def jsonify(data)
  data.json(@shell.indent)
end
```
```
jsonify(body)
jsonify(result)
jsonify(@shell.data(body))
```

Reply:
perhaps better suited for another PR..